lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -291,6 +291,23 @@ class EnumClass(Enum):
 
         assert get_hash(enum_1)
 
+    def test_different_enums(self):
+        class EnumClassA(Enum):
+            ENUM_1 = "hello"
+            ENUM_2 = "world"
```

Comment:
Since we're not actually doing anything with ENUM_2 here (or in EnumClassB), let's just remove?